app/controllers/application_controller.rb

```diff
@@ -6,5 +6,6 @@ class ApplicationController < ActionController::Base
 
   def configure_permitted_parameters
     devise_parameter_sanitizer.permit(:sign_up, keys: [:first_name, :last_name])
+    devise_parameter_sanitizer.permit(:account_update, keys: [:first_name, :last_name])
```

Comment:
why is this needed to account_update?

Reply:
when i try to update my account is shows me first name and last name unpermitted